addons/project/static/src/js/project_form.js

```diff
@@ -15,7 +15,7 @@ const ProjectFormController = FormController.extend({
         }
     },
     _onDomUpdated() {
-        const $editable = this.$el.find('.note-editable');
+        const $editable = this.$el.find('[name="description"]');
```

Comment:
You could probably keep the class name in the selector to be sure we continue to select the same element as before.